appengine/flexible/django_cloudsql/requirements.txt

```diff
@@ -1,5 +1,4 @@
-Django==2.0.3
-mysqlclient==1.3.12
+Django==1.11.13
```

Comment:
It may be better to specify the version based on the pip version with [requirement specifiers](http://pip.readthedocs.io/en/stable/reference/pip_install/#requirement-specifiers)

Reply:
This LGTM, but dpebot will probably swing through and update it again if we don't also change https://github.com/GoogleCloudPlatform/repository-gardener